src/dashboard/apps/consent/helpers.py

```diff
@@ -76,3 +83,65 @@ def send_mail(recipients: list, template_id: int, email_data: dict) -> int | Non
         # fail silently and send a sentry log
         sentry_sdk.capture_exception(e)
         return 0
+
+
+def duplicate_expiring_consents():
```

Comment:
In think we'd rather use the renew / renewal semantic. WDYT?

```suggestion
def renew_expiring_consents():
```